Source/Core/Core/HW/EXI/EXI_DeviceEthernet.cpp

```diff
@@ -155,6 +162,17 @@ void CEXIETHERNET::ImmWrite(u32 data, u32 size)
     {
     case INTERRUPT:
       exi_status.interrupt &= data ^ 0xff;
+      // raise back if there is still data
+      if (page_ptr(BBA_RRP) != page_ptr(BBA_RWP))
+      {
+        if (mBbaMem[BBA_IMR] & INT_R)
+        {
+          mBbaMem[BBA_IR] |= INT_R;
+
+          exi_status.interrupt |= exi_status.TRANSFER;
+        }
+      }
+
```

Comment:
This and a few other changes in this file look like bugfixes to the existing Ethernet device implementation, rather than directly part of your new built-in BBA implementation. Can you make a separate PR or at least a separate commit for this?